### PR TITLE
feat: adshield on ppss.kr

### PR DESCRIPTION
**Pull request by seia-soto**
Yet another method to block adshield over #260 .

### CODE REVIEW

**Comment by seia-soto** [comment]
Check first just like #260 did and merge if images are available.

**Comment by seia-soto** [comment]
Reverse engineering on javascript-obfuscator with any option doesn't take long time as they won't change the code execution order. Please, let me know if you get adblock error.

File: filter-uBO.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ! Title: List-KR uBO
 ! Description: List-KR for uBlock Origin
-! Version: 2022.324.10
+! Version: 2022.325.1
```
Comment (piquark6046) [inline filter-uBO.txt]: Please keep in mind what the version must be followed UTC timezone.

File: filter.txt [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ! Title: List-KR
 ! Description: List-KR for AdGuard
-! Version: 2022.324.11
+! Version: 2022.325.1
```
Comment (piquark6046) [inline filter.txt]: Please keep in mind what the version must be followed UTC timezone.

**Review by piquark6046 — CHANGES_REQUESTED** [review]
Your rule does not work, and cause anti-adblock detection of Ad-Shield.

<details>
<summary>Screenshot</summary>

Screenshot

</details>

**Comment by seia-soto** [comment]
Ok, confirmed. I'll check more as there were other filters enabled. Also, confirmed that the versioning follows UTC.

**Comment by seia-soto** [comment]
I am not getting anti-adblock detection with the following filters applied. May I get suggestion about filter environment?

User rules:

```css
ppss.kr###custom_html-2
ppss.kr###custom_html-3
ppss.kr##.ad-video
ppss.kr##.brand-margin
ppss.kr##div[onclick*="clickSdrcCheck"]
ppss.kr#%#//scriptlet('set-constant', 'Promise.prototype.catch', 'noopFunc')
```

<details>
<summary>Screenshot</summary>

image

</details>

**Comment by piquark6046** [comment]
My Development Environment:
AdGuard Browser Extension 4.0.141
Mozilla Firefox Dev 99.0b7
Ubuntu 21.10

Filters:
<details>
<summary>Details</summary>

AdGuard Tracking Protection Filer, AdGuard URL Tracking, AdGuard Social, AdGuard Annoyances filter and List-KR (custom)

**Review by green1052 — APPROVED** [review]
working in kiwi browser

and ungoogled chromium- 99.0.4844.82

**Comment by seia-soto** [comment]
Still in investigation.*

**Comment by seia-soto** [comment]
Fresh installed FF Dev 99.0b7 but unable to reproduce with same rule-sets enabled. I'll postpone the merging process to check if the network cache at any level or other points caused the problem.

**Review by piquark6046 — APPROVED** [review]
Okay. I approve your PR regardless of my reproduction.

**Comment by seia-soto** [comment]
> Okay. I approve your PR regardless of my reproduction.

Thanks. However, please revert at any time when users experiencing same show up.

**Pull request merged**